modules/oxide/src/less/theme/components/toolbar-button/toolbar-select-button.less

```diff
@@ -37,4 +38,13 @@
     white-space: nowrap;
     width: @toolbar-button-bespoke-label-max-width;
   }
+
+  // Correct specific styles for split button on touch devices
+  // See toolbar-split-button.less
+  &.tox-platform-touch {
+    .tox-split-button .tox-tbtn.tox-tbtn--select.tox-tbtn--select {
+      padding: 0 (@toolbar-button-select-padding - @toolbar-split-button-chevron-touch-padding);
+      width: auto;
+    }
+  }
```

Comment:
This really seems like the wrong place for this. It should be inside `toolbar-split-button.less` since the button is still a split button, not a regular one.

Reply:
Additionally, a better solution I believe would be to make the existing rule not match instead. For example adding `:not(.tox-tbtn--select)`:
```
  // Increase the chevron size and decrease the button size on touch devices
  &.tox-platform-touch {
    .tox-split-button .tox-tbtn:not(.tox-tbtn--select):first-child {
      width: @toolbar-button-width - @toolbar-split-button-chevron-touch-padding;
    }

    .tox-split-button__chevron {
      width: @toolbar-button-chevron-width + @toolbar-split-button-chevron-touch-padding;
    }
  }
```

Reply:
Agree on both cases. I did not initially took into account existence of toolbar select buttons on their own, outside of split buttons.